test/dotcom_web/controllers/stop_controller_test.exs

```diff
@@ -6,10 +6,36 @@ defmodule DotcomWeb.StopControllerTest do
   alias DotcomWeb.StopController
   alias Routes.Route
   alias Stops.Stop
+  alias Test.Support.Factories
+  alias Test.Support.FactoryHelpers
+  alias Test.Support.Generators
   alias Util.Breadcrumb
 
   setup :verify_on_exit!
 
+  @banner_alert_effects_active_or_future [
+    :dock_closure,
+    :dock_issue,
+    :service_change,
+    :shuttle,
+    :station_closure,
+    :station_issue,
+    :stop_closure,
+    :stop_moved,
+    :stop_shoveling,
+    :suspension
+  ]
+
+  @banner_alert_effects_active_only [
+    :access_issue,
+    :elevator_closure
+  ]
+
+  @banner_alert_effects @banner_alert_effects_active_only ++
+                          @banner_alert_effects_active_or_future
```

Comment:
non-blocking: I wonder if some of the logic and attributes used in this PR can be moved to `Alerts.Stop` or some new module for increased clarity. Also it'd be nice to avoid duplicating these lists.